mantidimaging/gui/widgets/spectrum_widgets/roi_form_widget.py

```diff
@@ -37,3 +42,164 @@ def set_binning_visibility(self) -> None:
         self.bin_size_spinBox.setHidden(hide_binning)
         self.bin_step_label.setHidden(hide_binning)
         self.bin_step_spinBox.setHidden(hide_binning)
+
+
+class ROIPropertiesTableWidget(BaseWidget):
```

Comment:
Can we add a class docstring here with a brief overview of what is contained in the widget so we can quickly see what this is at a glance through IntelliSense without navigating to the class to see what is is.